src/Phinx/Db/Table/Column.php

```diff
@@ -76,37 +76,37 @@ class Column
     /**
      * @var mixed|null
      */
-    protected $default;
+    protected $default = null;
```

Comment:
Thats not necessary, no value is null.

Reply:
Alright, was not sure as some parameters had an explicit `null` set versus others which just had the typedoc. Can remove them throughout the codebase?

Reply:
The code sniffer (at least mine does^^) should remove the overhead `= null` usually. 
We might want to add e.g. https://github.com/spryker/code-sniffer/blob/master/Spryker/Sniffs/Classes/PropertyDefaultValueSniff.php on top